websites/B/BeatLeader/presence.ts

```diff
@@ -14,124 +14,218 @@ const presence = new Presence({
 		"https://cdn.rcd.gg/PreMiD/websites/B/BeatLeader/assets/81.gif", //EarthDay
 		`https://i.imgur.com${"/Kf8bXNc.gif"}`, //Christmas 2022
 		`https://i.imgur.com${"/4dpoXYa.gif"}`, //Christmas 2023
-	];
+	],
+	contexts: Record<string, string> = {
+		beatleader: "https://i.joerkig.com/87jjwa.png",
+		nomods: "https://i.joerkig.com/5mjrdr.png",
+		nopause: "https://i.joerkig.com/dip2wp.png",
+		golf: "https://i.joerkig.com/skx0k8.png",
+		scpm: "https://i.joerkig.com/ouhq3b.png",
```

Comment:
Assets should be 512x512px

Reply:
Forgor to resize will do once I'm home from work